### ISSUE TITLE
'ORGANIZATION' entity is not correctly identified

### ISSUE THREAD

**Issue by Anusha-12** [microsoft/presidio-research#96]
I tired the evaluation script to validate the synthetic_data present in the data directory. The evaluation score is NaN for 'ORGANIZATION', it is not able to correctly predict the output
<img width="742" alt="Screenshot 2024-01-08 at 1 33 13 PM" src="https://github.com/microsoft/presidio-research/assets/32312467/c21907ac-b17d-4bce-aca5-03a67a8af927">

`

**Comment by omri374** [comment]
Thanks, we'll look into this. Apologies for the delayed response.

**Comment by Anusha-12** [comment]
Hi, is there any update with this?

**Comment by omri374** [comment]
Apologies for the delay. Which evaluation script are you running? and with which model/nlp engine?
I tried to run it as is (on the first 300 samples in the synthetic dataset, and this is what I got:

![image](https://github.com/microsoft/presidio-research/assets/3776619/3475a027-45b6-4024-b083-e5e2978c7595)

![image](https://github.com/microsoft/presidio-research/assets/3776619/c3a667fb-81ca-49d6-8a0b-515e68c4249e)

![image](https://github.com/microsoft/presidio-research/assets/3776619/1c8207ef-f20b-4822-9a35-7208a81a942b)

If you're getting other results, please share a reproducible example and make sure you're using the latest code in the repo.

**Comment by Anusha-12** [comment]
yes, I am using the Presidio Analyzer with 'Jean-Baptiste/roberta-large-ner-english' and I got different results. I will upload the graph

**Comment by omri374** [comment]
I tried to reproduce with this model. This is the configuration I used:
```py

from presidio_analyzer import AnalyzerEngine
from presidio_analyzer.nlp_engine import TransformersNlpEngine, NerModelConfiguration


# Here we define a transformers based NLP engine, 
# but you can use this cell to customize your Presidio Analyzer instance

# Define which model to use
model_config = [{"lang_code": "en", "model_name": {
    "spacy": "en_core_web_sm",  # use a small spaCy model for lemmas, tokens etc.
    "transformers": "Jean-Baptiste/roberta-large-ner-english"
    }
}]


# Map transformers model labels to Presidio's
model_to_presidio_entity_mapping = dict(
    PER="PERSON",
    LOC= "LOCATION",
    ORG="ORGANIZATION"
)

ner_model_configuration = NerModelConfiguration(labels_to_ignore = ["O"], 
                                                model_to_presidio_entity_mapping=model_to_presidio_entity_mapping)

nlp_engine = TransformersNlpEngine(models=model_config,
                                   ner_model_configuration=ner_model_configuration)

# Set up the engine, loads the NLP module (spaCy model by default) 
# and other PII recognizers
analyzer_engine = AnalyzerEngine(nlp_engine=nlp_engine)

```

Results did show ORGANIZATION metrics, but please try to reproduce too and let me know if you were able to get this working.

<img width="921" alt="image" src="https://github.com/microsoft/presidio-research/assets/3776619/40923eac-7ac7-4188-881b-077e1a97b81e">
